### PR TITLE
Send facility specific messages via notification

### PR DESCRIPTION
**Pull request by Apurva-Nagar**
Allows sending of messages to subscribed users of a facility via notifications.
Button
Modal

### CODE REVIEW

**Comment by Apurva-Nagar** [comment]
@bodhish should the notify button be shown for all users or some specific user roles?

File: webpack.config.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ const OptimizeCssAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 const safePostCssParser = require("postcss-safe-parser");
 
 const prodPlugins = (isDev) => {
-  if (isDev) {
-    return [];
-  }
+  // if (isDev) {
+  //   return [];
+  // }
```
Comment (bodhish) [inline webpack.config.js]: @Apurva-Nagar can you revert it 
```suggestion
  if (isDev) {
    return [];
  }
```

**Review by bodhish — CHANGES_REQUESTED** [review]

**Comment by nihal467** [comment]
https://user-images.githubusercontent.com/57055998/123461198-6c4c0000-d606-11eb-9268-f264378de6e9.mp4

@Apurva-Nagar  test failed, 

1. the notification can be sent but once I use the function, I can't close the screen for going back to my dashboard
2. there is a text "Type your text" already prefilled inside the box

File: src/Components/Facility/HospitalList.tsx [hunk 2]
```diff
@@ -351,6 +371,48 @@ const HospitalListPage = (props: any) => {
...
+                            </div>
+                            <div>
+                              <textarea
+                                id="NotifyModalMessageInput"
+                                className="h-32 w-full border p-2 shadow-lg"
+                                onChange={(e) =>
+                                  setNotifyMessage(e.target.value)
+                                }
+                              >
+                                Type your message...
```
Comment (bodhish) [inline src/Components/Facility/HospitalList.tsx:400]: This should be a placeholder

**Pull request closed without merging**